File: INICIANTE/JAVASCRIPT/1009.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var prompt = function(texto) { return lines.shift(); };

var nomev = prompt("NOMEV");
var salariof = parseFloat(prompt("SALARIOF"));
var vendasmes = parseFloat(prompt("VENDASMES"));
var salariofinal = salariof + vendasmes*0.15;

console.log("TOTAL = R$ " + salariofinal.toFixed(2));

File: INICIANTE/JAVA/1009.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String nome = sc.next();
        double salariof = sc.nextDouble();
        double vendasmes = sc.nextDouble();
        double salariofinal = salariof + (vendasmes*15 / 100);
        System.out.print(String.format("TOTAL = R$ %.2f" , salariofinal) + "\n");
    }
} 
